### ISSUE TITLE
ui5-panel A11y issue that role button div includes role heading div

### ISSUE THREAD

**Issue by mydavidzhang** [SAP/ui5-webcomponents#5606]
### **Bug Description**
SAP successfactor customer reported the implementation of ui5-panel has A11y issue that role button div includes role heading div, and they should be around.
We found SAP UI5 library has similar implementation. After testing we found  if a heading is present within a "button" tag it will not be recognized as a heading because the button role takes over. 
We compared WC library (https://sap.github.io/ui5-webcomponents/playground/components/Panel/) vs W3C (https://w3c.github.io/aria-practices/examples/accordion/accordion.html). And found first one could not be recognized as heading, while second one could.
Please help investigate with high priority since it was reported by big customer. 

### **Expected Behavior**
role button div and role heading div should be around for ui5-panel.

### **Steps to Reproduce**
1. Go to https://sap.github.io/ui5-webcomponents/playground/components/Panel/
2. Inspect panel with collapse/expand button

### **Isolated Example**
https://sap.github.io/ui5-webcomponents/playground/components/Panel/

### **Context**
- UI5 Web Components version: 1.6.0

### **Log Output / Stack Trace / Screenshots**
![image (13)](https://user-images.githubusercontent.com/10460331/182211031-ce8efd83-2dbc-412f-b3ad-4ffd698d449a.png)
![image (14)](https://user-images.githubusercontent.com/10460331/182211053-d27d7e4d-9142-44e2-a83a-cd0e8c27249c.png)

<img width="1310" alt="Screen Shot 2022-08-01 at 10 13 53 AM" src="https://user-images.githubusercontent.com/10460331/182211149-b70292ca-3dfb-47e0-8cff-9a2d47b02e86.png">

### **Priority**
- [ ] Low
- [] Medium
- [x ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: SAP successfactor
- Business impact: Please help investigate with high priority since it was reported by big customer.

**Comment by MapTo0** [comment]
Hello @SAP/ui5-webcomponents-topic-rl ,

As we discussed with @elenastoyanovaa 
I am forwarding that to the queue.

Regards,
Martin

**Comment by mydavidzhang** [comment]
Close this issue because it's duplicated with https://github.com/SAP/ui5-webcomponents/issues/5504. We will track progress on that ticket.
